.github/workflows/render_nanomaps.yml

```diff
@@ -35,11 +35,6 @@ jobs:
         git commit -m "NanoMap Auto-Update (`date`)" -a || true
         git push -f -u origin nanomap-render
 
-    - name: 'Push Maps'
-      uses: ad-m/github-push-action@master
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-
```

Comment:
Maps are pushed in the step above this one on line 36, so this step is redundant entirely.